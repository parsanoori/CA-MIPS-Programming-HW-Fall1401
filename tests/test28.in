7
98249673
147836730
559759286
217334975
692467884
861491300
48050374