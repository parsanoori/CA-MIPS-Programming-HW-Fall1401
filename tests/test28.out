48050374
98249673
147836730
217334975
559759286
692467884
861491300